### DISCUSSION TITLE
osu.ppy.sh beatmap links in lazer now open in browser

### DISCUSSION THREAD

**Discussion by jojopipe** [ppy/osu#21838] [Q&A] [answered]
so, now that lazer uses the lazer.ppy.sh website it doesn't open beatmap links of osu.ppy.sh inside the game anymore.

![image](https://user-images.githubusercontent.com/78083515/209719135-00d96abd-647b-47a9-a174-df17663ec4e2.png)

![osu_2022-12-27_21-25-57](https://user-images.githubusercontent.com/78083515/209719107-3ad024d2-59a4-44f9-87de-88c91ce69af1.png)

any workarounds?

**Comment by LittleEndu** [comment]
I guess bots that want to work on both stable and lazer can switch to using the `osu` protocol links, example `osu://b/574471`

**Comment by peppy** [comment, marked as answer]
https://github.com/ppy/osu/pull/21854
